### DISCUSSION TITLE
Migrating a new column example?

### DISCUSSION THREAD

**Discussion by bencleary** [uptrace/bun#485] [Q&A] [answered]
Firstly I really like Bun! I am experimenting with migrations, coming from Django there is the command makemigrations which auto generates the migration based on a diff of the model, now I'm not asking for this but rather are there examples (go based, sql works fine) of incremental migrations? Just adding a new column for example? I have it working but it seems a little odd that it's not driven from the model.

**Comment by vmihailenco** [comment, marked as answer]
I've improved the example we have at https://github.com/uptrace/bun/tree/master/example/migrate

>it seems a little odd that it's not driven from the model.

We have https://github.com/uptrace/bun/issues/456 but no timeframe when that will be available.
